src/sentry/services/hybrid_cloud/filter_query.py

```diff
@@ -46,6 +49,7 @@ def serialize_many(
     ) -> List[OpaqueSerializedResponse]:
         pass
 
+    @rpc_method
```

Comment:
@mikejihbe @corps I think this matches the intent of `FilterQueryInterface`, but wanted to be sure.

The effect is that `serialize_many` and `get_many` are automatically included as remotely callable methods on `UserService` and whatever else extends this interface. AFAICT, this is intended and necessary because serializing requires additional database queries from the control silo, so we have no choice but to do it over the wire if it's called on the region silo.